javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,10 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  var names = [];
+  for (var i = 0; i < students.length; i++) {
+    for (var key in students[i]) {
+      names.push(students[i][key]);
+    }
```

Comment:
What if there was more than one property in the object? You would be better off getting the value of the `name` property directly.